notebooks/concepts/imviz_color_display.ipynb

```diff
@@ -0,0 +1,260 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f21afb11-3f9f-4f21-8301-7c108d4db7fb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from echo import delay_callback\n",
+    "from glue.core.component_link import ComponentLink\n",
+    "from matplotlib import cm\n",
+    "from skimage.io import imread\n",
+    "\n",
+    "from jdaviz import Imviz"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "699da430-3ecc-4a74-ad08-3fc5b6b567a5",
+   "metadata": {},
+   "source": [
+    "Read in the RGB layers as different Numpy arrays."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "03856648-38ac-4819-b5e7-aeacf4f5fbbd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Point filename to your RGB file (e.g., JPEG)\n",
+    "filename = 'ogle_galaxy_in_color.jpg'\n",
+    "\n",
+    "im = imread(filename)\n",
+    "im_r = im[:, :, 0][::-1, :]\n",
+    "im_g = im[:, :, 1][::-1, :]\n",
+    "im_b = im[:, :, 2][::-1, :]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bad85e72-c882-4886-b8b2-f424fd034858",
+   "metadata": {},
+   "source": [
+    "Start Imviz app and load RGB channels into different data layers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5d0f5574-807a-4952-ba48-aae7d693860a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imviz = Imviz()\n",
+    "imviz.load_data(im_r, data_label='Red')\n",
+    "imviz.load_data(im_g, data_label='Green')\n",
+    "imviz.load_data(im_b, data_label='Blue')\n",
+    "\n",
+    "viewer = imviz.app.get_viewer('viewer-1')\n",
+    "\n",
+    "# Manually link the 3 channels by pixels\n",
+    "refdata = viewer.state.reference_data\n",
+    "ids0 = refdata.pixel_component_ids\n",
+    "for data in imviz.app.data_collection[1:]:\n",
+    "    ids1 = data.pixel_component_ids\n",
+    "    imviz.app.data_collection.add_link(ComponentLink([refdata.id[ids0[0]]], data.id[ids1[0]]))\n",
+    "    imviz.app.data_collection.add_link(ComponentLink([refdata.id[ids0[1]]], data.id[ids1[1]]))\n",
+    "\n",
+    "# Because linking happens after load, the image display is broken a little.\n",
+    "# So, please do this manually **after** running this cell.\n",
+    "#\n",
+    "# 1. Uncheck both data from Data menu.\n",
+    "# 2. Re-check both data from Data menu.\n",
+    "\n",
+    "imviz.app"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29286689-1936-452b-98c3-a52c7e06b408",
+   "metadata": {},
+   "source": [
+    "Assign the different RGB layers matching colormaps."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "39982c0c-0482-407c-8e57-c5a0029d0537",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viewer.state.layers[0].cmap = cm.get_cmap('Reds_r')\n",
+    "viewer.state.layers[1].cmap = cm.get_cmap('Greens_r')\n",
+    "viewer.state.layers[2].cmap = cm.get_cmap('Blues_r')"
```

Comment:
```suggestion
    "viewer.state.color_mode = 'One color per layer'\n",
    "viewer.state.layers[0].color = 'red'\n",
    "viewer.state.layers[1].color = 'green'\n",
    "viewer.state.layers[2].color = 'blue'"
```

See also [color_mode](https://docs.glueviz.org/en/latest/api/glue.viewers.image.state.ImageViewerState.html#glue.viewers.image.state.ImageViewerState.color_mode) and [color](https://docs.glueviz.org/en/latest/api/glue.viewers.image.state.ImageLayerState.html#glue.viewers.image.state.ImageLayerState.color)

The real power of this mode is that one can also easily make e.g. 4- or 5- color images and so on. The colors can be any valid matplotlib color.

(Note that I've edited this as a suggestion, so this is untested code, hopefully I haven't broken the notebook syntax)

Reply:
Interesting! How does that work together with cmap, or are they mutually exclusive?

Reply:
Thanks, the `.color` really simplified things! But the result still doesn't look quite the same as original JPEG (attached in original post). What am I missing?

![Screenshot 2021-09-08 103033](https://user-images.githubusercontent.com/2090236/132528828-63a95aee-20a6-4531-ba4a-6b1d7ffd6d31.jpg)

Reply:
Maybe the limits need to be set to 0 to 255 for all three layers rather than using auto levels?

The cmap and one colour per layer modes are mutually exclusive as they require different alpha compositing approaches.

Reply:
Doing this doesn't change much.

```python
for i in range(3):
    viewer.state.layers[i].v_min = 0
    viewer.state.layers[i].v_max = 255
```

Reply:
Is there some scaling I am supposed to do with the array before displaying it?

```python
>>> print(im_r.min(), im_r.max())
0 250
>>> print(im_g.min(), im_g.max())
0 235
>>> print(im_b.min(), im_b.max())
0 155
```

Reply:
No scaling needed but I'll need to check if we should be using different bias/contrast values. What are the current defaults?

Reply:
I think current bias default is 0.5 and contrast 1.

Reply:
I'll try and look into it soon but I will need to focus on some of the higher priority PRs first

Reply:
No worries. This is not MVP.

Reply:
The original jpg uses the lupton rgb color scheme